x-pack/plugins/security_solution/public/assistant/get_comments/index.tsx

```diff
@@ -105,20 +102,7 @@ export const getComments = ({
         message.timestamp.length === 0 ? new Date().toLocaleString() : message.timestamp
       ),
       username: isUser ? i18n.YOU : i18n.ASSISTANT,
-      ...(message.isError
-        ? {
-            eventColor: 'danger',
-            css: css`
-              .euiCommentEvent {
-                border: 1px solid ${tint(euiThemeVars.euiColorDanger, 0.75)};
-              }
-              .euiCommentEvent__header {
-                padding: 0 !important;
-                border-block-end: 1px solid ${tint(euiThemeVars.euiColorDanger, 0.75)};
-              }
-            `,
-          }
-        : {}),
+      eventColor: message.isError ? 'danger' : undefined,
```

Comment:
@stephmilovic FYI that `EuiCommentEvent` respecting `eventColor` for all types is now fixed in EUI, and the workaround you added in https://github.com/elastic/kibana/pull/167674 should no longer be necessary - but please feel free to QA and let us know if that's not the case!

Reply:
amazing, thank you!!